documentation/3.1.0/dockerized/envs/interface_env

```diff
@@ -1,15 +1,18 @@
 API_DOC_URL=/elevate-interface/api-doc
 APPLICATION_ENV=development
 APPLICATION_PORT=3569
-INSTALLED_PACKAGES=elevate-user elevate-mentoring elevate-scheduler elevate-project elevate-entity-management elevate-self-creation-portal elevate-survey
+INSTALLED_PACKAGES=elevate-user elevate-mentoring elevate-scheduler
 MENTORING_SERVICE_BASE_URL=http://mentoring:3000
 NOTIFICATION_SERVICE_BASE_URL=http://notification:3002
 RATE_LIMITER_ENABLED=true
+RATE_LIMITER_GENERAL_LIMIT=50
 RATE_LIMITER_NUMBER_OF_PROXIES=0
-REQUIRED_PACKAGES=elevate-user@1.1.71 elevate-mentoring@1.1.56 elevate-scheduler@1.0.4
+RATE_LIMITER_PUBLIC_LOW_LIMIT=5
+REQUIRED_BASE_PACKAGES=user,mentoring,scheduler
```

Comment:
this should be
`  "REQUIRED_BASE_PACKAGES": "elevate-user elevate-mentoring elevate-scheduler",
`
right?

Reply:
in dev env i found this 
@nevil-mathew

Reply:
Lets stick to QA envs for now